lib/data/serializers/memo_execution_serializer.dart

```diff
@@ -1,88 +1,79 @@
-import 'package:memo/core/faults/errors/serialization_error.dart';
-import 'package:memo/data/serializers/memo_block_serializer.dart';
+import 'package:memo/data/serializers/memo_difficulty_parser.dart';
 import 'package:memo/data/serializers/serializer.dart';
-import 'package:memo/domain/enums/memo_difficulty.dart';
 import 'package:memo/domain/models/memo_execution.dart';
 
-class MemoExecutionSerializer implements Serializer<MemoExecution, Map<String, dynamic>> {
-  final blockSerializer = MemoBlockSerializer();
+class MemoExecutionKeys {
+  static const started = 'started';
+  static const finished = 'finished';
+  static const rawQuestion = 'question';
+  static const rawAnswer = 'answer';
+  static const markedDifficulty = 'markedDifficulty';
+}
 
+class MemoExecutionSerializer implements Serializer<MemoExecution, Map<String, dynamic>> {
   @override
   MemoExecution from(Map<String, dynamic> json) {
-    final rawStarted = json['started'] as int;
+    final rawStarted = json[MemoExecutionKeys.started] as int;
     final started = DateTime.fromMillisecondsSinceEpoch(rawStarted, isUtc: true);
 
-    final rawFinished = json['finished'] as int;
+    final rawFinished = json[MemoExecutionKeys.finished] as int;
     final finished = DateTime.fromMillisecondsSinceEpoch(rawFinished, isUtc: true);
 
-    final rawAnswer = json['answer'] as List;
-    final rawQuestion = json['question'] as List;
-
     // Casting just to make sure, because sembast returns an ImmutableList<dynamic>
```

Comment:
It's not referencing, it's just a comment on why this cast is necessary. Otherwise, I would need to create a serializer for the serializer just for the mere sake of sembast implementing a particular list

Reply:
I'm not saying to create another layer to abstract sembast 😅 you could just rewrite the comment like 

`Casting just to make sure, because the serialized list may be an ImmutableList<dynamic>`

Reply:
But the only reason we do this is because of sembast, no reason to simply ignore this fact just because we don't want a comment of a possible implementation in the data layer